core/templates/pages/topic-viewer-page/topic-viewer-page.component.ts

```diff
@@ -67,12 +68,13 @@ export class TopicViewerPageComponent implements OnInit {
     private urlInterpolationService: UrlInterpolationService,
     private urlService: UrlService,
     private windowDimensionsService: WindowDimensionsService,
+    private windowRef: WindowRef,
   ) {}
 
   ngOnInit(): void {
-    if (this.urlService.getPathname().endsWith('revision')) {
+    if (this.urlService.getHash() === 'revision') {
```

Comment:
Hi @Rijuta-s, we should not be changing this because it affects SEO negatively. We should maintain the current URL subpart to maintain uniformity with the pages that users navigate to from the selected tab. For example, visiting `/revision` and clicking on one of the revision cards will direct the use to `/revision/<subtopic-url-fragment>`. If the url fragment is incorrect, the page redirects to `/revision`. I would suggest fixing the URL fragment rather than introducing /chaning hash values for this functionality.

Reply:
Done. Added a video in proof section.